webrtc/errors.md

```diff
@@ -0,0 +1,15 @@
+# Errors
```

Comment:
I think this was copied over from MFA. Feel free to update it with some WebRTC content, or just remove it if it's not necessary right now

Reply:
yep good catch. just updated